odkx-src/docs-tech-guide.rst

```diff
@@ -250,68 +250,6 @@ on any computer.
          You will also need to be familiar with
          the relevant package manager for your system.
 
-   .. admonition: Understanding terminal commands
```

Comment:
Why was this chunk deleted?

Reply:
it was actually not showing up on the web as a far as I can tell... so the thought was we did not need it.

Reply:
If someone (like @EstherPhebe) wants to fix and update that section I am not opposed to it, it simply was confusing to me since it wasn't rendering.

Reply:
Is this info that someone contributing to docs is likely to need? It doesn't seem ODK-X specific aside from a bit of a description on how we present things, but I think that is intuitive enough. So okay deleting unless  @EstherPhebe thinks this would be good do add back (does it need updating? How much?)

Reply:
@wbrunette @elmps2018 Hello, the reason its not showing up on the web is because the admonition directive has only one colon instead of two. 
`.. admonition:: Understanding terminal commands`

I tried it locally and it worked. I think it can be left there since it gives an idea on how terminal commands are presented in the docs. It is not "necessary" because I think people can and have found their way around without it but its like a nice-to-have in the docs.

![image](https://user-images.githubusercontent.com/46350156/133787685-f1c80baa-974d-47f0-8f5c-fb63da348789.png)